conbench/entities/commit.py

```diff
@@ -54,6 +58,7 @@ class Commit(Base, EntityMixin):
     timestamp: Mapped[Optional[datetime]] = Nullable(s.DateTime(timezone=False))
 
     def get_parent_commit(self):
+        # Hm -- should this not be done with a foreign key relationship?
```

Comment:
I think this used to be a lot more important while traversing the git graph, but it's not really used anymore. So I don't think we need to try to optimize it. We might even get rid of this method.